java/pom.xml

```diff
@@ -26,162 +31,18 @@
         </license>
     </licenses>
 
-    <repositories>
-        <repository>
-            <id>confluent</id>
-            <url>http://packages.confluent.io/maven/</url>
-        </repository>
-    </repositories>
-
-    <properties>
-        <java.version>1.7</java.version>
-        <zookeeper.version>3.4.6</zookeeper.version>
-        <zkclient.version>0.8</zkclient.version>
-        <kafka.version>1.0.0-SNAPSHOT</kafka.version>
-        <kafka.scala.version>2.11</kafka.scala.version>
-        <assertj.version>2.2.0</assertj.version>
-        <junit.version>4.12</junit.version>
-        <apacheda.version>1.0.0-M33</apacheda.version>
-        <apacheds.version>2.0.0-M21</apacheds.version>
-        <bcpkix.version>1.54</bcpkix.version>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <argparse4j.version>0.7.0</argparse4j.version>
-        <slf4j-api.version>1.7.6</slf4j-api.version>
-        <gson.version>2.7</gson.version>
-    </properties>
-
     <dependencies>
         <dependency>
-            <groupId>com.101tec</groupId>
-            <artifactId>zkclient</artifactId>
-            <version>${zkclient.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.zookeeper</groupId>
-            <artifactId>zookeeper</artifactId>
-            <version>${zookeeper.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.kafka</groupId>
-            <artifactId>kafka-clients</artifactId>
-            <version>${kafka.version}</version>
-        </dependency>
-        <dependency>
-            <groupId>com.google.code.gson</groupId>
-            <artifactId>gson</artifactId>
-            <version>${gson.version}</version>
-            <scope>compile</scope>
-        </dependency>
-        <dependency>
-            <groupId>net.sourceforge.argparse4j</groupId>
-            <artifactId>argparse4j</artifactId>
-            <version>${argparse4j.version}</version>
+            <groupId>io.confluent</groupId>
+            <artifactId>utility-belt</artifactId>
+            <version>${project.version}</version>
         </dependency>
+
         <dependency>
             <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
+            <artifactId>slf4j-simple</artifactId>
```

Comment:
Is this right? Seems like it would change the logging behavior. are we using this instead of the slf4j-log4j we use everywhere else for some reason? Also, is it going to affect the logging output format, which could be a compatibility issue?

Reply:
It's weird, but this is what I arrived at after testing the images. The logger doesn't seem to work with -api alone.

I added the -api dependency to common-docker and removed it here, but that cause the image to default to a noop logger, as opposed to the official image's behavior of logging verbosely. I added the -api dependency back in directly, and logging was still busted with a clean build. I stackoverflowed as hard as I could, and saw that others solved this problem by depending on the -simple artifact instead, which seemed to work. I am guessing something was getting pulled in transitively before, but stopped after the move to common-docker.